src/styles/getTooltipStyles.js

```diff
@@ -49,6 +49,31 @@ function computeHorizontalShift(windowWidth, xOffset, componentWidth, tooltipWid
     return 0;
 }
 
+/**
+ *
+ * @param {Number} [maxWidth] - Max-width for tooltip's wrapper.
+ * @param {Number} [tooltipTextWidth] -  Max-width for tooltip's wrapper.
+ * @returns {Object}
+ */
+function setWrapperWidth(maxWidth, tooltipTextWidth) {
+    if (!maxWidth) {
+        return {};
+    }
+
+    if (tooltipTextWidth >= maxWidth) {
+        return {maxWidth};
+    }
+
+    // Sum left and right wrapper padding, otherwise the wrapper will clip the text.
+    // Sometimes the real width of a text is a value with fractional digits (i.e. 172.2px)
+    // but the text.offsetWidth returns a truncated integer (172px) making this function
+    // to set the wrapper width slightly smaller than text but enough to make the last word
+    // of a line to wrap or clip. So is neccesary add 1px extra to the sum so wrapper contains text entirely.
```

Comment:
```suggestion
    // Add horizontal padding to the textContainerWidth to get the wrapper width. tooltipTextWidth ignores the fractions (OffsetWidth) so add 1px to fit the text properly. 
```

How does this sound?

Reply:
```suggestion
    // Add horizontal padding to tooltipTextWidth to get the wrapper width. 
    tooltipTextWidth ignores the fractions (OffsetWidth) so add 1px to fit the text properly. 
```
Sounds nice, I would change the word `textContainerWidth` to `tooltipTextWidth`. `textContainerWidth` sounds like another variable.

Reply:
You can always use normal text instead of technical variables